fili-core/src/main/java/com/yahoo/bard/webservice/web/endpoints/JobsServlet.java

```diff
@@ -415,11 +423,12 @@ protected void handleResultsResponse(
      * return an Observable wrapping the PreResponse as is.
      *
      * @param preResponse  The PreResponse to be inspected
+     * @param jobsApiRequest  JobsApiRequest object with all the associated info with it
      *
      * @return An Observable wrapping the PreResponse or an Observable wrapping a ResponseException
      *
      */
-    protected Observable<PreResponse> handlePreResponseWithError(PreResponse preResponse) {
+    protected Observable<PreResponse> handlePreResponseWithError(PreResponse preResponse, ApiRequest jobsApiRequest) {
```

Comment:
If there's specific pieces of ApiRequest we keep needing to have, we can extract an interface from it and depend on that instead if it makes more sense. Not sure if it would help here or not, but it's an option.